string task_id
string robot_id
string picker_id
string initiator_id
string responder_id
string state

